user/design/ms-iis-framework-component.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: user-doc
+title: IIS Framework Component
+---
+
+The _iis-framework_ [component](./components.html) is related to multiple Windows/.Net related packs used to configure how
```

Comment:
Not true. I think it is the component that manages the configuration for Microsoft IIS ... also link to its website